src/openapi/openapi.json

```diff
@@ -6128,6 +6128,134 @@
         "x-router-controller": "cors"
       }
     },
+    "/api/data/workflowcreate/step": {
+      "post": {
+        "tags": [
+          "data"
+        ],
+        "summary": "Create step",
+        "description": "Returns status of step creatiom",
+        "operationId": "createStep",
+        "requestBody": {
+          "x-name": "payload",
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/Workflow"
+              }
+            }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "OK",
+            "headers": {
+              "Access-Control-Allow-Origin": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "Access-Control-Allow-Methods": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "Access-Control-Allow-Headers": {
+                "schema": {
+                  "type": "string"
+                }
+              }
+            },
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Workflow"
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "Bad request"
+          }
+        },
+        "security": [
+          {
+            "Cognito_Authorizer": ["openid"]
+          }
+        ],
+        "x-amazon-apigateway-integration": {
+          "uri": "${data_lambda_arn}",
+          "passthroughBehavior": "when_no_match",
+          "httpMethod": "POST",
+          "type": "aws",
+          "requestTemplates": {
+            "application/json": "#set( $req = $input.params()) {\"resource\":\"workflow\",\"operation\":\"createStep\",\"params\":{\"data\": \"$req.path.get('data')\", \"type\": \"$req.path.get('type')\", \"step_name\": \"$req.path.get('step_name')\" },\"context\":{\"user_id\": \"$context.authorizer.claims.sub\", \"user_email\": \"$context.authorizer.claims.email\"}}"
```

Comment:
While attributes like 
```
step_status_label
action_id
form_id
service_id
```
aren't required arguments, these should be optional arguments of this endpoint.